### PR TITLE
Homepage redesign, new folder for partial templates

### PR DESCRIPTION
**Pull request by dannypernik**

### CODE REVIEW

**Comment by nickolashe** [comment]
@dannypernik Should we review this together sometime? Probably makes sense to begin with resolving the conflicts.

File: openCurrents/templates/create-event.html [hunk 1]
```diff
@@ -119,6 +119,13 @@ <h6 class="fit-left navy"> at </h6>
 
           <a id="add-location"><i class="fa fa-plus-circle half-margin-right"></i>Add another location</a>
 
+          <div class="row three-halves-margin-top">
+            <input type="radio" id="event-privacy-1" name="event-privacy-[]" value="1" class="hidden styled-radio" checked/>
```
Comment (nickolashe) [inline openCurrents/templates/create-event.html]: @karbmk @dannypernik lets make sure this is working properly. i don't think we are using the `-[]` syntax anymore

Comment (nickolashe) [inline openCurrents/templates/create-event.html]: this entire section is implemented differently i believe

Comment (nickolashe) [inline openCurrents/templates/create-event.html]: https://github.com/opencurrents/opencurrents/issues/333

**Comment by nickolashe** [comment]
@dannypernik can you try merging https://github.com/opencurrents/opencurrents/pull/344 and see what the result is (since both need to be merged for the new changes)

File: openCurrents/forms.py [hunk 2]
```diff
@@ -217,6 +217,15 @@ def __init__(self, *args, **kwargs):
             'placeholder': 'do some good'
         })
     )
+
+    CHOICES = [(True, ''), (False, '')]
```
Comment (nickolashe) [inline openCurrents/forms.py]: @dannypernik @karbmk guys were both of you typing at the same time?! `is_public` element appears twice in this form.

Comment (nickolashe) [inline openCurrents/forms.py]: furthermore, no matter which `is_public` def i sue, the form throws an exception upon submitting. we need to resolve this before merging

Comment (karbmk) [inline openCurrents/forms.py]: That's weird. I will look on it.

File: openCurrents/views.py [hunk 3]
```diff
@@ -956,7 +956,7 @@ def form_valid(self, form):
             )
         else:
             self.project = None
-        if str(self.request.POST['event-privacy-[]']) == '1':
+        if str(self.request.POST['is_public']) == '1':
```
Comment (nickolashe) [inline openCurrents/views.py]: @karbmk is there a reason why we are accessing request.POST directly vs. through `form.cleaned_data`?

**Comment by dannypernik** [comment]
screen shot 2017-08-22 at 12 29 21 am

@nickolashe I've merged #344 and #353 into this PR. Orgs and OrgUsers are being created when /nonprofit and /business forms are being submitted. However, when I confirm my account and set a password, I'm sent to the org-signup page with org name displayed as status message. We should direct straight to /profile with some status message or popup welcoming them.

Also, perhaps this is taken care of in another PR, but everything looks good with admin-profile until I try to confirm hours in approve-hours.

**Comment by nickolashe** [comment]
@dannypernik what happens when you try approving hours?

**Comment by dannypernik** [comment]
screen shot 2017-08-22 at 11 43 16 am

@nickolashe

**Comment by karbmk** [comment]
@dannypernik , I think it's because it doesn't have the latest change which is in develop. I think you should pull views file and approve-hours template to this branch!

**Comment by dannypernik** [comment]
@karbmk cool thanks! I got it working. Opened issue #356 after pulling latest changes.

**Pull request merged**